crates/sui-snapshot/src/writer.rs

```diff
@@ -303,32 +312,17 @@ impl StateSnapshotWriterV1 {
         let join_handle = tokio::spawn(async move {
             let results: Vec<Result<(), anyhow::Error>> = ReceiverStream::new(receiver)
                 .map(|file_metadata| {
-                    let backoff = backoff::ExponentialBackoff::default();
                     let file_path = file_metadata.file_path(&epoch_dir);
                     let remote_object_store = remote_object_store.clone();
                     let local_object_store = local_object_store.clone();
                     let local_dir_path = local_dir_path.clone();
                     async move {
-                        retry(backoff, || async {
```

Comment:
I assume this retry was for the issue you mentioned in the PR summary, and hence we no longer need retries?

Reply:
Ah no, actually the reason we no longer need retries is that we refactored `copy_file` since and the retries are included as part of `copy_file` itself.